docs/administration/recent_activity/recent_activity.md

```diff
@@ -40,7 +40,7 @@ It can be limited to "Only own log".
 The "Activity Log / Read" policy should be given to every roles having access to the Back Office,
 at least with the "Only own log" owner limitation,
 to allow them to use the "Recent activity" block in the default dashboard or their custom dashboard.
-This policy is also required by user having a profile to properly view their own.
+This policy is also required by user having a profile to view this profile with its activity log.
```

Comment:
Personally, I think the order of statements should be reversed.

```suggestion
This policy is required to view activity log in user profile, if profile is enabled.
```

This way the statements go from most important parts first, into a description that clarifies the conditions.

Reply:
Done in 30d4fd1d3caf4fc64520e35f3590bafd016516a5